algobot/assets.py

```diff
@@ -884,18 +884,25 @@
 \x00\x00\x01\x75\x1f\xa3\x93\x96\
 "
 
-qt_version = [int(v) for v in QtCore.qVersion().split('.')]
+qt_version = [int(v) for v in QtCore.qVersion().split(".")]
```

Comment:
We shouldn't really touch this file as it's generated by QT itself

Reply:
Ahh okays - we can ignore it if needed